extensions/opentelemetry/opentelemetry/deployment/src/main/java/io/quarkus/opentelemetry/deployment/OpenTelemetryBuildItem.java

```diff
@@ -0,0 +1,17 @@
+package io.quarkus.opentelemetry.deployment;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.quarkus.builder.item.SimpleBuildItem;
+import io.quarkus.runtime.RuntimeValue;
+
+public final class OpenTelemetryBuildItem extends SimpleBuildItem {
```

Comment:
This is used on a producer side, but not being consumed.

Is it needed?

Reply:
Ah, I guess I added it to use it as a parameter to the Vert.x Tracer, but then I ended up using `GlobalOpenTelemetry.get()`. I'll remove it.